stvs/SageTV7/SageTV7.xml

```diff
@@ -67182,7 +67182,7 @@
           </Action>
           <Action Name="&quot;REM Option for: Configure Placeshifter Connection&quot;" Sym="OPUS4A-210962">
            <Conditional Name="!IsEmbeddedSystem()" Sym="OPUS4A-211270">
-            <Conditional Name="( IsServerUI() || (!IsDesktopUI() &amp;&amp; IsRemoteUI()) || (IsLinuxOS() &amp;&amp; GetServerProperty(&quot;linux/enable_nas&quot;,false)) ) &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
+            <Conditional Name="IsLocalConnection() &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
```

Comment:
well, since I've got to make changes anyway, I can just revert out the commits, and resubmit them under the 'right' account (strange, because I was pretty sure git was logging in as CraziFuzzy).  Either way, I've got to get it sorted in my git config, so I'll just do that.